app/assets/javascripts/bastion/components/bst-alert.directive.js

```diff
@@ -10,16 +13,25 @@
  *     <div bst-alert="success"></div>
  *   </pre>
  */
-angular.module('Bastion.components').directive('bstAlert', function () {
+angular.module('Bastion.components').directive('bstAlert', ['$animate', '$timeout', function ($animate, $timeout) {
+    var SUCCESS_FADEOUT = 5000;
```

Comment:
Is this the standard timing and/or congruent with Foreman fade outs? Feels like forever to me, however, I was starring at it, daring it to fade away.